.github/workflows/people.yml

```diff
@@ -53,6 +53,35 @@ jobs:
             exit 1
           fi
 
+  build-contact:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: src/frontend/app/contact
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+      - name: Install yarn
+        run: npm install -g yarn
+      - name: Restore cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            .next/cache
+          # Generate a new cache whenever packages or source files change.
+          key: ${{ runner.os }}-nextjs-${{ hashFiles('**/yarn.lock') }}-${{ hashFiles('**.[jt]s', '**.[jt]sx') }}
+          # If source files changed but packages didn't, rebuild from a prior cache.
+          restore-keys: |
+            ${{ runner.os }}-nextjs-${{ hashFiles('**/yarn.lock') }}-
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Build App Contact
+        run: yarn build
+
   build-mails:
```

Comment:
are you ready to have a lint-front job?

Reply:
The build job seems to do a lint check, so if the lint fail the build job will fail (https://github.com/numerique-gouv/people/actions/runs/7445795989/job/20254761158?pr=3). 

Do you still want to have the `lint-front` job so ?

Reply:
I can see the job check-changelog is required, do I start to add things inside the changelog or is it a mistake ?

Reply:
no it's required because you can now disable it by adding the "noChangeLog" label to your PR.